translator/tocwconfig/tocwconfig_test.go

```diff
@@ -50,7 +51,13 @@ func TestBaseContainerInsightsConfig(t *testing.T) {
 	context.CurrentContext().SetRunInContainer(true)
 	t.Setenv(config.HOST_NAME, "host_name_from_env")
 	t.Setenv(config.HOST_IP, "127.0.0.1")
-	expectedEnvVars := map[string]string{}
+	t.Setenv(envconfig.AWS_CA_BUNDLE, "sampleConfig/public_amazon_cert.pem")
+	defer func() {
```

Comment:
Can we just call `os.Clearenv()` in `resetContext()` so we cleanup all env vars in a common place. Not sure if any tests will break due to this but worth a try.

Reply:
`t.Setenv()` should be unique to the test executed, which I think is better than trying to set with `os.Setenv` and clearing that.